pkg/cloud/libvirt/client/domain.go

```diff
@@ -426,6 +426,8 @@ func domainDefInit(domainDef *libvirtxml.Domain, name string, memory, vcpu int)
 		return fmt.Errorf("machine does not have an DomainVcpu set")
 	}
 
+	domainDef.CPU.Mode = "host-passthrough"
```

Comment:
host-model?

Reply:
I personally prefer host-passthrough. You don't plan migrating them, and host-passthrough is the most compatible one. The only downside is that it works well with KVM, may have less success with QEMU (but why would we need QEMU only virtualization?)

Reply:
I don't mind much.

Reply:
fine by me

Reply:
As i said in the issue, `host-passthrough` is pretty good but if you want the most reliable solution, you need to do a bit of extra work: Ask libvirt for host capabilities and set a custom cpu mode based on the best capability you see. Anyway, no objection on this change.